web/src/routes/backOffice/employees/components/EmployeeForm.svelte

```diff
@@ -26,10 +30,37 @@
 	 */
 	export let title: string;
 
+	/**
+	 * Set form as a create form, to create a new employee.\
+	 * @default false
+	 */
+	export let createForm: boolean = false;
```

Comment:
```suggestion
	export let action: "create" | "edit";
```
Instead of using a boolean, imo it would be better to have a union type